avocado/core/job.py

```diff
@@ -556,6 +563,8 @@ def run(self):
             self.log.error('Report bugs visiting %s', _NEW_ISSUE_LINK)
             return exit_codes.AVOCADO_FAIL
         finally:
+            if self.job_post_dispatcher.extensions:
+                self.job_post_dispatcher.map_method('run', self)
```

Comment:
Yes I know it could be, it's a matter of taste and I see some benefit in my proposal, but I can live with this version too.